events/message.js

```diff
@@ -29,13 +29,13 @@ module.exports = (client, message) => {
   // the bots user id (a mention).
   const prefixes = [settings.prefix, `<@!${client.user.id}>`];
   const prefix = prefixes.find(p => message.content.startsWith(p));
-  if (message.content.indexOf(prefix) !== 0) return;
+  if (message.content.toLowerCase().indexOf(prefix.toLowerCase()) !== 0) return;
```

Comment:
This code is completely wrong, line 32 always evaluated to true, I have no idea why that existed, but the check should have been:

```javascript
if (!prefix) return;
```

Either way, the correct way to do this is by doing this instead:

```javascript
const prefixes = [settings.prefix.toLowerCase(), `<@!${client.user.id}>`];

const content = message.content.toLowerCase();
const prefix = prefixes.find(p => content.startsWith(p));
if (!prefix) return;
```